lib/dialogs/settings/panels/display.jsx

```diff
@@ -104,6 +104,7 @@ const DisplayPanel = props => {
       >
         <Item title="Light" slug="light" />
         <Item title="Dark" slug="dark" />
+        <Item title="System" slug="system" />
```

Comment:
This will appear in the web app too. Do we care that it may or may not have any impact there?